acl/errors.go

```diff
@@ -126,3 +128,24 @@ func PermissionDeniedByACLUnnamed(_ Authorizer, context *AuthorizerContext, reso
 	desc := NewResourceDescriptor("", context)
 	return PermissionDeniedError{Accessor: "", Resource: resource, AccessLevel: accessLevel, ResourceID: desc}
 }
+
+func RequirePermissionDeniedError(t testing.TB, err error, _ Authorizer, _ *AuthorizerContext, resource Resource, accessLevel AccessLevel, resourceID string) {
```

Comment:
I don't think our production code should be importing `testing` and `testify` here. Could these be moved to test-specific files?